nucypher-core-wasm/src/lib.rs

```diff
@@ -542,6 +542,116 @@ impl EncryptedTreasureMap {
     }
 }
 
+//
+// Request Keys
+//
+
+#[wasm_bindgen]
+#[derive(derive_more::From, derive_more::AsRef)]
+pub struct RequestSharedSecret(nucypher_core::RequestSharedSecret);
+
+#[wasm_bindgen]
+#[derive(PartialEq, Eq, Debug, derive_more::From, derive_more::AsRef)]
+pub struct RequestPublicKey(nucypher_core::RequestPublicKey);
+
+#[wasm_bindgen]
+impl RequestPublicKey {
+    #[wasm_bindgen(js_name = fromBytes)]
+    pub fn from_bytes(data: &[u8]) -> Result<RequestPublicKey, Error> {
+        from_bytes::<_, nucypher_core::RequestPublicKey>(data)
+    }
+
+    #[wasm_bindgen(js_name = toBytes)]
+    pub fn to_bytes(&self) -> Box<[u8]> {
+        to_bytes(self)
+    }
+
+    #[allow(clippy::inherent_to_string)]
+    #[wasm_bindgen(js_name = toString)]
+    pub fn to_string(&self) -> String {
+        format!("{}", self.0)
+    }
+
+    pub fn equals(&self, other: &RequestPublicKey) -> bool {
+        self.0 == other.0
+    }
```

Comment:
Just curious, would this work?
```suggestion
    pub fn equals(&self, other: &RequestPublicKey) -> bool {
        self == other
    }
```